packages/carbon-component-mapper/src/files/radio.d.ts

```diff
@@ -0,0 +1,21 @@
+import { FormGroupProps } from "./form-group";
+import { UseFieldApiComponentConfig, AnyObject } from "@data-driven-forms/react-form-renderer";
+import { ReactNode } from "react";
+
+export interface RadioOption extends AnyObject {
+    value: any;
+    label?: ReactNode;
+}
+
+interface InternalRadioProps extends AnyObject {
+    isDisabled?: boolean;
+    label?: ReactNode;
+    options: RadioOption[];
+    FormGroupProps?: AnyObject;
```

Comment:
This should be the form group interface declared above.

Reply:
This is a little bit confusing, but FormGroup interface should be probably named “commonProps” because in this mapper there is no FormGroup component. (The name is generated via the generate command.) This prop is for Carbon’s FormGroup component.